sympy/geometry/line.py

```diff
@@ -2121,10 +2121,12 @@ def __new__(cls, p1, p2, **kwargs):
         p2 = Point(p2, dim=2)
         if p1 == p2:
             return p1
-        if (p1.x > p2.x) == True:
-            p1, p2 = p2, p1
-        elif (p1.x == p2.x) == True and (p1.y > p2.y) == True:
-            p1, p2 = p2, p1
+        dir = kwargs.get('dir', False)
```

Comment:
I'll do it once the issue with `area` is cleared up. `evaluate` seems a little strange at first but I guess it will be better since it's known to be used to keep args in order.